src/PackageCompiler.jl

```diff
@@ -483,6 +483,7 @@ function create_sysimage(packages::Union{Nothing, Symbol, Vector{String}, Vector
                          compat_level::String="major",
                          extra_precompiles::String = "",
                          )
+    get_compiler_cmd()
```

Comment:
Perhaps add a comment to explain why this is called in both locations but not used?

Reply:
Done!